### ISSUE TITLE
ID 258694: [Supporting the platform -Simon – Value Mapping]: “Set Color” Dialog is truncated when the page is set to viewport 320*256 Under "Value mapping" blade.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66495]
### **User Experience:**  
Users with low vision who depend on zooming functionality for better readability will be impacted as they will be not able to see partially visible and overlapped contents present on the page due to which they will miss valuable information which can block their further actions.
_Note: User credentials should NOT be included in the bug._

### **Pre-requisite:** 
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/) Sign in with valid credentials. "Home" page appears.  
2. Navigate the server requests pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate the page "Edit" page which appears. Navigate to the "All" buttons and select it. 
5. Navigate to "Value Mapping" button and expand it. 
6. Navigate all the controls present under it. Verify the issue

**Actual Result:** 
Issue: “Set Color” Dialog is truncated when the page is set to viewport 320*256 Under "Value mapping" blade. 
Observation: While navigating through 'Set color' Dialog' colors (Red, Orange) are partially visible.

**Expected Result:**
When the view port is set to 320*256 Set Color” Dialog should be properly visible and should not be truncated under Value mapping" blade.
_Note: Same issue exists with the graphs present under Home page._

### **Severity:**
`SEV2
`
### **Area:**
- [ ] **#grafana-frontend-platform**: `ValueMappingEditRow Component` / `ColorPicker Component` (Saga UI Design System)
Same issue exists with Explore->Inspector. Same issue exists with Explore->Inspector -.Stats>query->..>Error. 
- [ ] **#grafana-explore**: `Explore Pullup`

**Comment by L-M-K-B** [comment]
Besides of `ColorPicker` and `ValueMappingEditRow` this issue is also related `Modal` that's why I added a task in [this issue](https://github.com/grafana/grafana/issues/67302).

**Comment by ashharrison90** [comment]
still an issue

**Comment by yaelleC** [comment]
Pulled up some numbers quickly and last month we had 55 clicks on `inspect` from displays that had a width resolution 320 or under : query [here](https://ops.grafana-ops.net/explore?schemaVersion=1&panes=%7B%224xe%22:%7B%22datasource%22:%22_STwrCg4k%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22grafana-bigquery-datasource%22,%22uid%22:%22_STwrCg4k%22%7D,%22project%22:%22grafanalabs-global%22,%22location%22:%22%22,%22format%22:1,%22rawSql%22:%22SELECT%20%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3C%3D%20320%29%20as%20sub_320,%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3C%20320%20and%20context_screen_inner_width%20%3C%3D%20820%29%20as%20sub_820,%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3E%20820%20and%20context_screen_inner_width%20%3C%3D%201280%29%20as%20between_820_and_1280,%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3E%201280%20and%20context_screen_inner_width%20%3C%201512%29%20as%20between_1280_and_1512,%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3E%201512%20and%20context_screen_inner_width%20%3C%201920%29%20as%20between_1512_and_1920,%5Cn%20%20%20%20countif%28context_screen_inner_width%20%3E%3D%201920%29%20as%20over_1920,%5CnFROM%20%60grafanalabs-global.rudderstack.grafana_panel_inspect_json_panel_data_clicked%60%20%5CnWHERE%20$__timeFilter%28cast%28received_at%20as%20timestamp%29%29%20%5Cn%5Cn%22,%22editorMode%22:%22code%22,%22sql%22:%7B%22columns%22:%5B%7B%22type%22:%22function%22,%22parameters%22:%5B%5D%7D%5D,%22groupBy%22:%5B%7B%22type%22:%22groupBy%22,%22property%22:%7B%22type%22:%22string%22%7D%7D%5D,%22limit%22:50%7D,%22rawQuery%22:true%7D%5D,%22range%22:%7B%22from%22:%22now-30d%22,%22to%22:%22now%22%7D%7D%7D&orgId=1)
